content/applications/finance/accounting/taxation/taxes/taxcloud.rst

```diff
@@ -43,6 +43,46 @@ In Odoo
   and the zip code). Go to :menuselection:`Settings --> Users & Companies --> Companies`
   to open and edit your Company record.
 
+Automatically post taxes to the correct Tax Payable account
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* For each one of your companies that uses TaxCloud, it is necessary to create a 
+  **User-Defined Default** so that the new taxes generated by the TaxCloud integration are created 
+  with the correct Tax Payable account:
```

Comment:
```suggestion
* For each one of your companies that uses TaxCloud, it might be necessary to create a 
  **User-Defined Default** so that the new taxes generated by the TaxCloud integration are created 
  with the correct Tax Payable account:
  
.. warning::

    User defined defaults impacts all records at creation.  It means that **All* new taxes will target the specified Tax 
    Payable account, unless the default value is modified manually (or another default takes precedence over the one 
    you  defined).
```

Necessary might be too strict IMHO, I suppose some database still work fine without it :D.

Also, please add a clear explanation on "User defined defaults" to make sure the user knows this change impacts not only the taxcloud taxes (my explanation can be changed, but I believe the information should be provided).

Reply:
Hi @Feyensv I agree and thanks for your edits! I've slightly rephrased them while keeping your meaning, do feel free to let me know if there's anything you'd like me to change!

Reply:
LGTM, only remaining step is to update your commit message to have a clear title & message according to our [guidelines](https://www.odoo.com/documentation/14.0/developer/misc/other/guidelines.html#commit-message-structure) .
We do not strictly respect all points, but the commits should still follow the global logic:
* title:
  * prefix: `[ADD]` if new page and content, `[IMP]` if improvement of existing content, `[REM]` in case of content removal
  * targeted scope: the scope/files modified by the commit (keep it simple, but we need to know what's targeted at first glance)
  * short explanation of the change or content improvement
* Detailed explanation in commit message, if useful.  Sometimes the commit title is enough to understand the commit content.

Something like:
```
[IMP] accounting/taxcloud: how to specify default account for new taxes

Detailed explanation
```

You can reuse the explanation you specified in the PR description as commit message ;).

Reply:
Thank you so much Victor for explaining this to me! I have changed the commit message to conform to the guidelines. :)